docs/tutorial/tut_point_in_time.md

```diff
@@ -1,37 +1,48 @@
-A Point-In-Time table is one of two business-vault query-helper tables (the other being Bridge tables) designed for loading and creating the presentation marts.
```

Comment:
ALL Metadata in this file needs to align with the metadata reference. i.e. NOT defined in `dbt_project.yml` but YAML string blocks or in-line.

Reply:
Resolved